app/models/organization_webhook.rb

```diff
@@ -57,7 +57,7 @@ def admin_org_hook_scoped_github_client
   #
   # Returns true if successful, otherwise raises a GitHub::Error or ActiveRecord::RecordInvalid.
   def create_org_hook!(client)
-    github_organization(client).create_organization_webhook(config: { url: webhook_url }).id
+    self.github_id = github_organization(client).create_organization_webhook(config: { url: webhook_url }).id
```

Comment:
The `self.github_id` was previously missing meaning this method only created webhooks and didn't save their IDs.